doc/sphinx/practical-tools/coqide.rst

```diff
@@ -327,3 +327,123 @@ If you choose something else than UTF-8, then missing characters will
 be written encoded by `\x{....}` or `\x{........}` where each dot is
 an hexadecimal digit: the number between braces is the hexadecimal
 Unicode index for the missing character.
+
+.. _coqide-debugger:
+
+Debugger
+--------
+
+Version 8.14 introduces a preliminary visual debugger for Ltac tactics within
+CoqIDE.  It supports setting breakpoints visually, automatically
+displaying the stopping point in the source code with "continue",
+"step over" "step in" and "step out" operations.  It does not yet
+display the call stack or variable values.
+
+The debugger is based on the :ref:`Ltac debugger <interactive-debugger>`.
+We'd like to eventually support other scripting facilities such as Ltac2,
+but that will be addressed later.  (Since Ltac2 shares tactic evaluation
+logic with Ltac, it may already work to some extent.)
+
+Even though the debugger is incomplete, we believe it can already provide
+some value to users.  You may encounter bugs or usability issues.  The behavior
+and user interface will evolve as the debugger is completed.  Please feel free
+to suggest changes and improvements.
+
+GUI
+~~~
+
+This screenshot shows the debugger stopped at a breakpoint in the Ltac tactic
+`my_tac`.  Breakpoints are shown with a red background and the stopping point is
+shown with a dark blue background.  `Set Ltac Debugger.` enables stopping in the
```

Comment:
Does not work with https://github.com/coq/coq/pull/14252
![image](https://user-images.githubusercontent.com/396076/118325715-dc2c7e00-b4d1-11eb-8cf2-fd5db607d807.png)

Reply:
Please open an issue.  This PR is for documentation.

You're brave trying this on such a (I think) large and complex script.  You're the first person to report on their experience using the debugger.  It may be challenging to get reasonable-sized examples that reproduce problems.

Reply:
And no performance measurement or tuning yet.

Reply:
I think the issue is just 
```suggestion
shown with a dark blue background.  `Set Ltac Debug.` enables stopping in the
```
unless in fact you meant to write "Debugger"

Reply:
Indeed, a typo.  Got it.